Update requirements.txt to resolve installation issues

IndigoDosSantos opened the pull request:
@exx8 
- Problem: The previous requirements.txt file caused installation errors.
- Solution: This pull request includes a revised requirements.txt file that addresses the installation issues. I've tested it to ensure successful installation, but only on my computer.
- Additional Changes: The README.md has been updated with the installation command for PyTorch-cu118.

IndigoDosSantos committed:
Update requirements.txt

Original `requirements.txt` didn't work for me.

IndigoDosSantos committed:
Update requirements.txt

IndigoDosSantos committed:
Update requirements.txt

IndigoDosSantos committed:
Create install.bat

IndigoDosSantos committed:
Update install.bat

IndigoDosSantos committed:
Create install.sh

IndigoDosSantos committed:
Update README.md

Added necessary PyTorch install

exx8 commented:
First of all thank you twice!
First for the feedback and second for the PR.
I will review it soon.
Thank you again!

exx8 commented:
Sorry for the delay.
First the requirements.txt you suggest seems to be better,
and the install scripts are a great addition!
I am not sure why we need torchaudio, also IF requires SentencePiece, so I think to add it to the requirements.
What do you think? Thanks!
Again sorry for the delay it was very busy weekend.

IndigoDosSantos committed:
Update install.sh

Deleted `torchaudio`

IndigoDosSantos committed:
Deleted `torchaudio`

IndigoDosSantos committed:
Update requirements.txt

Add `sentencepiece`, which is required for DeepFloyd/IF

IndigoDosSantos committed:
Merge pull request #1 from EtienneDosSantos/patch-2

Patch 2

IndigoDosSantos committed:
Update README.md

Deleted `torchaudio` as it is not needed

IndigoDosSantos commented:
Thx for the review! You're right; `sentencepiece` is necessary for IF. I've added it to 'requirements.txt' and removed the nonsensical `torchaudio` from both installation scripts. 😅

After committing these minor changes, I tested all models. Everything worked fine except for IF. In Stage 3, IF loads 'stabilityai/stable-diffusion-x4-upscaler/unet/diffusion_pytorch_model.bin', which my antivirus flagged as infected with 'Trojan.Ciusky.CryptedAit.1'. It's also odd that `diffusers` chose this file over the available safetensors variant, as it usually prefers safetensors. While this could be a false positive, it's definitely best to err on the side of caution and not use IF until this is sorted out.

exx8 commented:
Hi,

I'm sorry for the delayed response. I'm currently going through a demanding period.

If it's okay, I will use your suggestions in the future once I have the capacity to maintain the repo.

Thanks for your patience.

IndigoDosSantos commented:
Hey :)

No worries at all! I can assure you there is zero pressure from my side 🤗
